lib/ansible/modules/cloud/amazon/ec2_asg_facts.py

```diff
@@ -265,6 +294,7 @@ def find_asgs(conn, module, name=None, tags=None):
                         "protected_from_scale_in": false
                     }
                 ],
+                "launch_config_name": "public-webapp-production-1",
                 "launch_configuration_name": "public-webapp-production-1",
```

Comment:
Did you add launch_config_name in addition to launch_configuration_name to the return just to be more consistent with ec2_asg?